src/modules/hide_sidebar_elements/index.js

```diff
@@ -8,7 +8,8 @@ class HideSidebarElementsModule {
             id: 'hideFeaturedChannels',
             name: 'Hide Recommended Channels',
             defaultValue: true,
-            description: 'The left sidebar is too cluttered, so you can remove recommended channels'
+            description:
```

Comment:
do you have an autoformatter enabled? can you undo all these changes?

Reply:
Yes and yes. I usually disable prettier, sorry.